src/core/lib/channel/metrics.cc

```diff
@@ -17,14 +17,25 @@
 #include "src/core/lib/channel/metrics.h"
 
 namespace grpc_core {
+namespace {
+// Uses the Construct-on-First-Use idiom to avoid the static initialization
+// order fiasco.
+std::vector<GlobalInstrumentsRegistry::GlobalInstrumentDescriptor>& Get() {
```

Comment:
Now that this is no longer a member of the class, it should probably be called something like `GetInstrumentList()`.

Reply:
Done.